web/src/entities/Operator/Operator.service.ts

```diff
@@ -0,0 +1,13 @@
+import {CompareOperator, CompareOperatorNameMap, CompareOperatorSymbolMap} from './Operator.constants';
```

Comment:
Moving forward services will be static functions that provide an object with methods and attributes. This allows the possibility of composing if required.